frontend/src/App.js

```diff
@@ -19,9 +19,9 @@ import ArticlesIndexPage from "main/pages/Articles/ArticlesIndexPage";
 import ArticlesCreatePage from "main/pages/Articles/ArticlesCreatePage";
 import ArticlesEditPage from "main/pages/Articles/ArticlesEditPage";
 
-import PlaceholderIndexPage from "main/pages/Placeholder/PlaceholderIndexPage";
-import PlaceholderCreatePage from "main/pages/Placeholder/PlaceholderCreatePage";
-import PlaceholderEditPage from "main/pages/Placeholder/PlaceholderEditPage";
+import HelpRequestIndexPage from "main/pages/HelpRequest/HelpRequestIndexPage";
+import PlaceholderCreatePage from "main/pages/HelpRequest/PlaceholderCreatePage";
```

Comment:
Lines 23/24 should be /HelpRequest/HelpRequest[Page_Type]Page. You still have placeholder

Reply:
Thanks. I made the mistake of doing Index before i fixed all the placeholders, meaning I only changed it for index